Cannot find module 'babelify'

Hello, 

my project setup uses browserify and no babel.

looking at the vue-select it should render es2015 and therefore no need for babel.

i've installed the 2.2.0 version from npm

any guidance is very welcome.

FYI, I've been running into this same issue when I try adding in vue-select from NPM packages.
My setup is also using browserify+vueify _without_ babel

`$ npm run watch`
`> blacksmith@1.0.0 watch C:\Main\Git-Repo\blacksmith`
`> watchify -t vueify -e src/main/js/main.js -o src/main/resources/public/scripts/blacksmith.js -v`
`Error: Cannot find module 'babelify' from 'C:\Main\Git-Repo\blacksmith\node_modules\vue-select'`

The error message would certainly suggest that it's related to vue-select.
No issues when using via CDN however.

Still seeing this issue as well in latest NPM install. My build setup in Grunt and Browserify. 
```
Running "browserify:dev" (browserify) task
>> Error: Cannot find module 'babelify' from '/Users/seanthompson/Tagboard/tgb-embed/node_modules/vue-select'
```

--- a/package.json
+++ b/package.json
@@ -15,12 +15,6 @@
     "test": "karma start test/unit/karma.conf.js --single-run",
     "test-watch": "karma start test/unit/karma.conf.js --single-run=false --auto-watch=true"
   },
-  "browserify": {
-    "transform": [
-      "vueify",
-      "babelify"
-    ]
-  },
   "repository": {
     "type": "git",
     "url": "https://github.com/sagalbot/vue-select.git"

